var/spack/repos/builtin/packages/iqtree2/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class Iqtree2(CMakePackage):
+    """Efficient and versatile phylogenomic software by maximum likelihood"""
+
+    homepage = "http://www.iqtree.org"
+    url = "https://github.com/iqtree/iqtree2/archive/refs/tags/v2.1.2.tar.gz"
+
+    version("2.1.2", sha256="3aaf5ac7f60d852ac8b733fb82832c049ca48b7203a6a865e99c5af359fcca5a")
+
+    depends_on("boost", type=("build", "run"), when="@2:")
+    depends_on("eigen", type=("build", "run"), when="@1.6:")
+    depends_on("zlib", type=("build", "run"))
```

Comment:
I highly doubt any of these dep types are correct. Build/run means to put `prefix.bin` in `PATH`, but none of these even come with executables.

Reply:
so should they just be build dependencies?

Reply:
They should just be link dependencies if there are no exes